src/content/events/2023/bytesize_nextflow_executors.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Bytesize: Nextflow Executors'
+subtitle: Maxime Garcia, Seqera
+type: talk
+start_date: '2023-12-05'
+start_time: '13:00+02:00'
```

Comment:
```suggestion
start_time: '13:00+01:00'
```
I guess?

Reply:
so quick with your comment. But yes, I already changed it.